roles/openshift_health_checker/openshift_checks/package_version.py

```diff
@@ -78,8 +80,10 @@ def get_required_ovs_version(self):
         Else ensure Open vSwitch version 2.4"""
         openshift_version = self.get_openshift_version()
 
-        if float(openshift_version) < 3.5:
+        if _version_lt(openshift_version, "3.5"):
             return self.openshift_to_ovs_version["3.4"]
+        if _version_lt("3.7", openshift_version):
+            return self.openshift_to_ovs_version["3.7"]
```

Comment:
To implement the suggestion, we could stop using strings for the version keys, use tuple of ints instead, and then:

```python
openshift_to_ovs_version = {
    (3, 7): [...],
    ...
}

...

latest = max(openshift_to_ovs_version)
earliest = min(openshift_to_ovs_version)

if version > latest:
    return self.openshift_to_ovs_version[latest]

if version < (3, 0):
    raise ...

if version < earliest:
    return self.openshift_to_ovs_version[earliest]

return ...